### ISSUE TITLE
Assertain if the gear damage coeff need to match the gear coeff and fix

### ISSUE THREAD

**Issue by wlbragg** [c172p-team/c172p#1292]
See https://github.com/c172p-team/c172p/pull/1289#issuecomment-516582789 and https://github.com/c172p-team/c172p/pull/1289#issuecomment-517371539

**Comment by dany93** [comment]
I've done a few tests **with the Default gear**, by restoring the coefficients in `c172p-damage.xml`
nose 3600 / 1200
main 5400 / 1800
(same as those in the FDM)

At these quick tests (to be checked more carefully, by me and someone else) I think that the coefficients can be restored in the damage file, to be consistent.
- The gears break reasonably on hard landings, not the wings.
- I could break the nose gear alone by landing with the nose down.
- The nose gear did not break at full trust, at rest with the brakes and yoke at full pitch down, even with the 180 hp engine.

I noticed that "Repair" was inoperative if the engine was still running (at rest on the ground, nose gear not broken). Not a big issue, can be circumvented but a bit surprising.

**Comment by wlbragg** [comment]
>I noticed that "Repair" was inoperative

Yes, I don't know why we opted to do that, I think it was for realism as in you probably wouldn't or couldn't fix your aircraft with the engine running or while in the air.
I remember conceding to that because it could be circumvented by cycling the "Allow Damage" choice.

Thanks for testing the coeff in the damage file. I'll do the same and see if I get similar findings.

**Comment by onox** [comment]
> I think it was for realism as in you probably wouldn't or couldn't fix your aircraft with the engine running or while in the air

That was indeed as far as I remember. But you can probably still turn off damage and then turn it on again.
